### ISSUE TITLE
TypeError: sequence index must be integer, not 'str'

### ISSUE THREAD

**Issue by tetianakh** [pytest-dev/pytest-cov#144]
Hello,

I get this error when one of the tests fails:
```
Error processing line 1 of /path/to/lib/python2.7/site-packages/pytest-cov.pth:
  Traceback (most recent call last):
    File "/path/to/lib/python2.7/site.py", line 152, in addpackage
      exec line
    File "<string>", line 1, in <module>
    File "<string>", line 10, in <module>
  TypeError: sequence index must be integer, not 'str'
Remainder of file ignored
```

Without coverage, I just get a normal pytest output with traceback for the exception that was raised in the failing test.

This happens in Travis CI builds, so in case you want to have a look, here's the link: https://travis-ci.org/tetianakh/bcbio-nextgen/builds/189138489
Output from the same test without coverage: https://travis-ci.org/tetianakh/bcbio-nextgen/builds/189126412

Any ideas what this might be about?

**Comment by ionelmc** [comment]
Also, the error is about failure to setup coverage for subprocess (a feature of pytest-cov, and the main selling point). If it happens it means something weird is going on in the process (usually pytest_cov or coverage could not be imported).

**Comment by shaunduncan** [comment]
Hi, I was having a similar issue and stumbled across this report. I have noticed that the pth fix here addresses the issue, at least for me: https://github.com/pytest-dev/pytest-cov/commit/8d22dcc0f8f9c6fe5c070f1061062c9e5e733952#diff-89a1ea490ef6f4c6c356c2ba7fc5f0df

Currently I have a workaround that's not exactly pretty, but it gets the job done. So I guess I was curious: are there any upcoming plans for a new release to be submitted to PyPI that includes the above mentioned commit?

Thanks!

**Comment by ionelmc** [comment]
Ooops. I forgot to make release. I'm preparing one today.

**Comment by shaunduncan** [comment]
Wonderful! Thank you so much, I'll keep an eye out for it!

**Comment by ionelmc** [comment]
Published `2.5.0`.
